src/main/java/app/entities/unitconverter/TemperatureConverter.java

```diff
@@ -0,0 +1,29 @@
+package app.entities.unitconverter;
+
+public class TemperatureConverter {
+
+    public double celsiusToFahrenheit(double celsius) {
+        return (celsius * 9/5) + 32;
+    }
+    public double celsiusToKelvin(double celsius) {
+        return celsius + 273.15;
+    }
+
```

Comment:
inconsistencies in newlines. Please use 1 newline between each method

Reply:
Fixed it, only 1 new line between each method